core/src/main/java/com/novoda/noplayer/internal/exoplayer/CompositeTrackSelector.java

```diff
@@ -19,7 +15,7 @@
 
 import java.util.List;
 
-class CompositeTrackSelector extends TrackSelector {
```

Comment:
We could probably extend the `DefaultTrackSelector` but it is also passed to the other track selectors 😬 I'll look at updating this again on dev, this is a quick fix to ensure that master works.

Reply:
it feels to me we are missing an interface somewhere